File: Publisher Integration/SPX Mediation/Unified Bidding | iOS Integration/Generic/Rewarded Video request.m
[SmaatoSDK prebidRewardedInterstitialForAdSpaceId:@"SMAATO_ADSPACE_ID"
                                   completion:^(SMAUbBid *_Nullable bid, NSError *_Nullable error) {
         
        if (bid && bid.metaData[@"smaato_ubid"]) {
            NSString *ubId = bid.metaData[@"smaato_ubid"];
            SMAAdRequestParams *requestParam = [[SMAAdRequestParams alloc] init];
            requestParam.ubUniqueId = ubId;
             
            [SmaatoSDK loadRewardedInterstitialForAdSpaceId:@"SMAATO_ADSPACE_ID" delegate:self requestParams:requestParam];
             
        } else {
            return;
        }
    }];

File: Publisher Integration/SPX Mediation/Unified Bidding | iOS Integration/Generic/Rewarded Video request.swift
SmaatoSDK.prebidRewardedInterstitial(forAdSpaceId: "SMAATO_ADSPACE_ID") { [unowned self] (ubBid, error) in
            guard let bid = ubBid,
                  let bidId = bid.metaData["smaato_ubid"] as? String else {
                return
            }
            let requestParams = SMAAdRequestParams()
            requestParams.ubUniqueId = bidId
            SmaatoSDK.loadRewardedInterstitial(forAdSpaceId: "SMAATO_ADSPACE_ID",
                                               delegate: self,
                                               requestParams: requestParams)
        }